File: relationship-manager/analytics-service/src/main/java/com/bank/rm/analytics/AnalyticsServiceApplication.java
package com.bank.rm.analytics;
import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
@SpringBootApplication(scanBasePackages = {"com.bank.rm.analytics", "com.bank.rm.common"})
public class AnalyticsServiceApplication {
    public static void main(String[] args) { SpringApplication.run(AnalyticsServiceApplication.class, args); }
}

File: relationship-manager/analytics-service/src/main/kotlin/com/bank/rm/analytics/AnalyticsServiceApplication.kt
package com.bank.rm.analytics

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication

@SpringBootApplication(scanBasePackages = ["com.bank.rm.analytics", "com.bank.rm.common"])
class AnalyticsServiceApplication

fun main(args: Array<String>) {
    runApplication<AnalyticsServiceApplication>(*args)
}
